recipe/meta.yaml

```diff
@@ -28,6 +27,9 @@ build:
     - export CXXFLAGS="${CXXFLAGS} -I/usr/include"
     #- export CUB_PATH=${PWD}/cub_v1.8.0  # TODO(leofang): remove this
     - "{{ PYTHON }} -m pip install . -vv"
+    # full CUDA 10.2 support was added in a commit post-7.2.0 release
+    # (d654524fa), cherry pick it and a prior commit (34d4dbd73) to support it
+    - git cherry-pick --strategy=recursive -Xtheirs -n 34d4dbd73eb534217095273499bfde03dc508d47 d654524fa1c5aae4df39090d148f5eadce4ebb2c
```

Comment:
To get patches from these, can just pull them from GitHub like so. Would add those here and then we can still use the tarball.

https://github.com/cupy/cupy/commit/34d4dbd73eb534217095273499bfde03dc508d47.patch
https://github.com/cupy/cupy/commit/d654524fa1c5aae4df39090d148f5eadce4ebb2c.patch